omnibus/files/private-chef-cookbooks/private-chef/recipes/nginx.rb

```diff
@@ -74,6 +72,13 @@
   node.default['private_chef']['nginx']['ssl_certificate_key'] = ssl_keyfile
 end
 
+# Link the required recipe file into the HTML static files root.
```

Comment:
Should we consider checking the permissions on the file provided by the user?  That is, should we require that the file has restricted permissions to avoid a situation where a random user on the machine can modify content served to all chef-clients?